Makefile

```diff
@@ -16,6 +16,17 @@ TARGET   := alertmanager
 
 include Makefile.COMMON
 
+REV        := $(shell git rev-parse --short HEAD 2> /dev/null  || echo 'unknown')
+BRANCH     := $(shell git rev-parse --abbrev-ref HEAD 2> /dev/null  || echo 'unknown')
+HOSTNAME   := $(shell hostname -f)
```

Comment:
FreeBSD man pages say it's the same as leenucks.
https://www.freebsd.org/cgi/man.cgi?query=hostname&sektion=1